cpp/src/io/parquet/ipc/Schema_generated.h

```diff
@@ -139,13 +139,13 @@ inline const MetadataVersion (&EnumValuesMetadataVersion())[5]
   return values;
 }
 
-inline const char* const* EnumNamesMetadataVersion()
+inline char const* const* EnumNamesMetadataVersion()
```

Comment:
This file has this comment on the top.
`// automatically generated by the FlatBuffers compiler, do not modify`
if it is possible to enforce east const with `FlatBuffers compiler`, please add that as comment in top of the file too.